Description: fix tests without network
 without $tcount, we get:
 skip() needs to know $how_many tests are in the block at t/10_quotes.t line 11
 remove duplicate use()es while we're here
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2015-05-12

--- a/t/10_quotes.t
+++ b/t/10_quotes.t
@@ -1,8 +1,6 @@
-use FindBin;
-use lib $FindBin::RealBin;
-use testload;
-
-use Test::More tests => 16;
+my $tcount;
+BEGIN { $tcount = 16 }
+use Test::More tests => $tcount;
 
 use FindBin;
 use lib $FindBin::RealBin;
